sponsorship_compassion/models/contracts.py

```diff
@@ -521,6 +523,18 @@ def _on_sponsorship_finished(self):
                 else:
                     contract.signal_workflow('contract_terminated')
 
+            # Create Sponsorship Message
+            message_obj = self.env['gmc.message.pool']
+            action_id = self.env.ref(
+                'sponsorship_compassion.cancel_sponsorship').id
+
+            logger.info("DELETE MESSAGE CREATED")
+            message_vals = {
+                'action_id': action_id,
+                'object_id': sponsorship.id

```

Comment:
you can add `'partner_id': sponsorship.correspondant_id.id, 'child_id': sponsorship.child_id.id`